PADAS/react-native-jsonforms-formatter
# Optimize build

## Summary
Optimize build to migrate from standard typescript to webpack.

## Fixes
#6 

## Proposed changes
- Move all tests out of src directory
  - Ensure tests are not included in build
- Build using webpack to dist directory
- Update Readme

## Test
- Run the following build phases and ensure all pass
  - `yarn clean`
  - `yarn build`
    - Confirm build output in dist folder
  - `yarn test`
  - `npm ci`
